Classes/Notifications/NotificationModelController.swift

```diff
@@ -183,8 +183,10 @@ final class NotificationModelController {
 
         githubClient.client.send(V3SubscribeThreadRequest(id: model.v3id, ignore: model.watching)) { result in
             switch result {
-            case .success: break
+            case .success:
+                Haptic.triggerSelection()
```

Comment:
Shouldn't this be `Haptic.triggerNotification(.success)`?

Reply:
That's what I wasn't sure about. I was more thinking about making it a bit transparent for user that there's an API call and the chance of failure exists. And only for scenarios that it fails, trigger the failure haptic. But open to the feedback, if you'd feel it'd better to be `notification`, I'm down to update the code

Reply:
Let's give it a try as-is